src/components/Score/index.module.scss

```
@@ -0,0 +1,39 @@
+.container{
```

Fijate de poner los espacios entre nombre de clase y { en todo el archivo:
```suggestion
.container {
```